x/examples/outline-fetch/main.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/Jigsaw-Code/outline-sdk/x/examples/internal/config"
+)
+
+func main() {
+	transportFlag := flag.String("transport", "", "Transport config")
+	flag.Parse()
+
+	url := flag.Arg(0)
+	if url == "" {
+		log.Fatal("Need to pass the URL to fetch in the command-line")
+	}
+
+	dialer, err := config.MakeStreamDialer(*transportFlag)
+	if err != nil {
+		log.Fatalf("Could not create dialer: %v", err)
+	}
+	httpClient := &http.Client{Transport: &http.Transport{DialContext: func(ctx context.Context, network, addr string) (net.Conn, error) {
```

Comment:
*nit* can we separate this into multiple lines? It looks a little bit messy. But if `go format` wraps it into a single line then just leave it as it is now.

Reply:
Cleaned up